### ISSUE TITLE
Improve experience of maintaining Dashboards in Beats and packages

### ISSUE THREAD

**Issue by ChrsMark** [elastic/beats#30932]
Dashboards maintained in https://github.com/elastic/integrations have been diverging from the ones maintained in this repo. Also using https://github.com/elastic/elastic-package to export Dashboards does not follow the same format with the tools in Beats.
It would be nice to have a more unified process so ease the maintaince of the Dashboards in both repos.
We can add a new flag to the developer scripts in beats, so integration dashboards can be transformed to the beats format [as suggested](https://elastic.slack.com/archives/CJZ9U5G9X/p1647862879374209).

**Comment by jlind23** [comment]
@nimarezainia what should we do there? Should we continue working on both dashboard format in parallel?

**Comment by nimarezainia** [comment]
@jlind23 not at the moment. Focus should be given to the https://github.com/elastic/integrations dashboards.
